lib/components/AlbumScreen/song_menu.dart

```diff
@@ -874,13 +988,28 @@ class PlaybackAction extends StatelessWidget {
       child: IconButton(
         icon: Column(
           children: [
-            Icon(
-              icon,
-              color: iconColor,
-              size: 32,
-              weight: 1.0,
+            SizedBox(
+              width: 35,
+              height: 46,
+              child: Stack(
+                alignment: Alignment.topCenter,
+                children: [
+                  Icon(
+                    icon,
+                    color: iconColor,
+                    size: 35,
+                    weight: 1.0,
+                  ),
+                  Positioned(
+                    bottom: -2,
+                    child: Text(
+                      value ?? "",
+                      style: Theme.of(context).textTheme.labelSmall,
+                    ),
+                  ),
+                ],
+              ),
```

Comment:
We probably don't need this value text anymore, right?

Reply:
I thought you maybe wanted to keep it, but I will remove it then.